timur/lib/client/jsx/components/query/ant_switch.tsx

```diff
@@ -0,0 +1,41 @@
+// From https://material-ui.com/components/switches/#CustomizedSwitches.tsx
+import {withStyles, Theme, createStyles} from '@material-ui/core/styles';
+import Switch from '@material-ui/core/Switch';
+
+const AntSwitch = withStyles((theme: Theme) =>
```

Comment:
Cool.  This is definitely another place where typescript will help -- going to the definition of withStyles and following the definitions around clarifies this api.